aptos-move/parallel-executor/src/proptest_types/types.rs

```diff
@@ -59,6 +61,31 @@ impl<K: Hash + Clone + Debug + Eq + PartialOrd> ModulePath for KeyType<K> {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Arbitrary)]
+pub struct ValueType<V: Into<Vec<u8>> + Debug + Clone + Eq + Arbitrary>(
```

Comment:
remove Into<Vec\<u8\>>

Reply:
It's needed, because the types used by testing have Into<Vec<u8>> which I use to grab u128, that's why I wrote extract_u128, to show how it will be used (for testing deltas in the PE fake testing framework)